relay-sampling/src/condition.rs

```diff
@@ -441,6 +441,7 @@ pub enum RuleCondition {
 
     /// An unsupported condition for future compatibility.
     #[serde(other)]
+    #[default]
```

Comment:
Does `GenericFilterConfig` need to be default (I don't think so). For future compatibility with global config, it might make sense to make the `condition` field for `GenericFilterConfig` optional, though.

Reply:
Good points! Will fix